src/main/java/com/faforever/client/game/TeamCardController.java

```diff
@@ -32,6 +35,8 @@
   public Pane teamPaneRoot;
   public VBox teamPane;
   public Label teamNameLabel;
+  private final HashMap<Integer, RatingChangedLabelController> ratingChange = new HashMap<Integer, RatingChangedLabelController>();
```

Comment:
* Final fields should be initialized in the constructor for the sake of consistency
* `ratingChange` sounds like a variable containing an integer. `ratingChangeLabelControllerByPlayerId` would describe it more clearly